src/Grid/GridHeader.js

```diff
@@ -110,144 +93,111 @@ class GridHeader extends React.Component {
       value2 = 1;
     }
     
-    this.filterHandler(value1, value2);
+    this.filterHandler(value1, value2, true);
   }
 
-  filterHandler = (value1, value2) => {
+  filterHandler = (value1, value2, hasFilter) => {
```

Comment:
Can we provide a more meaningful name for value1, value2? It is not clear what those are intended for